lib/routes.js

```diff
@@ -1,3 +1,9 @@
+
```

Comment:
This is important for JS compressors that don't delete a first comment aka banner in a file and that blank line may cause them to assume there is no banner.